app/app.py

```diff
@@ -1,10 +1,15 @@
 import logging
+import os
+from pathlib import Path
 
 from flask import Flask
 
 from app.config import Config
 
 
+LOGGING_DIR = Path(os.environ.get("LOGGING_DIR", ".data"))
```

Comment:
I don't have a strong pref either way but would this make more sense in config.py?

Reply:
Gah! Sorry I never got back on this - yes, it should probably go in config.py. I can do a follow up PR